### PR TITLE
upgrade decidim 0.17.1

### PR DESCRIPTION
**Pull request by isaacmg410**
This PR upgrade with the latest release of decicidm (0.17.1)
Fix Specs
Bump version.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -4,15 +4,9 @@ source 'https://rubygems.org'
...
-# Force gem rails to 5.2.2.1 to fix some vulnerabilities
-# on actionview and railties
-# It can be removed when new stable version will be released or
-# when Decidim force the rails version
-gem 'rails', '5.2.2.1'
-
 gemspec
 
-gem 'decidim', "~> 0.16.0"
+gem 'decidim', ">= 0.17.0"
```
Comment (tramuntanal) [inline Gemfile]: To maximize compatibility the change should be 
```suggestion
gem 'decidim', ">= 0.16.0"
```

File: Gemfile.lock [hunk 2]
```diff
@@ -1,62 +1,62 @@
 PATH
```
Comment (tramuntanal) [inline Gemfile.lock]: We should remove the Gemfile.lock file from source control as modules should not constrain the versions of the dependencies in the applications where they are used.

File: decidim-verifications-barcelona_energia_census.gemspec [hunk 3]
```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |s|
 
   s.files = Dir['{app,config,lib}/**/*', 'LICENSE-AGPLv3.txt', 'Rakefile', 'README.md']
 
-  DECIDIM_VERSION = '~> 0.16.0'
+  DECIDIM_VERSION = '>= 0.17.0'
```
Comment (tramuntanal) [inline decidim-verifications-barcelona_energia_census.gemspec]: Same as before
```suggestion
  DECIDIM_VERSION = '>= 0.16.0'
```

**Comment by isaacmg410** [comment]
@tramuntanal done!

**Pull request merged**